### PR TITLE
Feat/user role UI

### PR DESCRIPTION
**Pull request by dantgw**
- Added test cases for visibility of register button for admin and non-admin users
- Removed the progress bar only when pool has ended. (I tried to add test cases for progress bar as well, but ran into some issues.)

### CODE REVIEW

**Comment by ottodevs** [comment]
✅ **Deployment Complete!** 🌐
| Details | Link |
| --- | --- |
| **View Deployment** | [Visit Site](https://pwa-mcbe6p3x4-pool-team.vercel.app) |

**Review by ottodevs — CHANGES_REQUESTED** [review]
Looks good for now, just check if we need to use `cn` and we merge after this is clear.

Thanks for the good work!

**Comment by ottodevs** [comment]
✅ **Deployment Complete!** 🌐
| Details | Link |
| --- | --- |
| **View Deployment** | [Visit Site](https://pwa-bhr9rrow5-pool-team.vercel.app) |

File: src/app/pwa/(pages)/pool/new/page.tsx [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 import CreatePoolForm from './create-pool-form'
+import { cn } from '@/lib/utils/tailwind'
 
 export default async function CreatePoolPage() {
     const authStatus = await getAuthStatus()
     const isAuthorized = authStatus.isAuthenticated && authStatus.isAdmin
 
     if (!isAuthorized) {
-        return <div>You are not authorized to create a pool.</div>
+        return <div className={cn('mt-4 w-full text-center')}>You are not authorized to create a pool.</div>
```
[the remark this answers is not in the document]
Comment (dantgw) [reply src/app/pwa/(pages)/pool/new/page.tsx]: Okay, I removed it.

[the remark this answers is not in the document]
Comment (ottodevs) [reply src/app/pwa/(pages)/pool/new/page.tsx]: Thanks!

**Review by ottodevs — APPROVED** [review]
LGTM!

**Pull request merged**